provider/postgis/postgis.go

```diff
@@ -199,14 +221,49 @@ func NewTileProvider(config dict.Dicter) (provider.Tiler, error) {
 		return nil, err
 	}
 
+	// check if provider layer are automatically generated
+	var autoTables []string
+	if auto {
+		// get auto config settings
+		// *NOTE: I'm guessing this should be a map, not a map slice (we only want 1 auto config per provider) but couldn't get it to work
+		autoConfig, err := config.MapSlice(ConfigKeyAutoConfig)
+
+		if err != nil {
+			return nil, err
+		}
+		// generate layers from information_schema of provider
+		autoTables, err = genLayers(p.pool)
+		for i := 0; i < len(autoTables); i++ {
+
+			// this is very hacky. I was having problems creating an empty dict.Dicter
+			defaultLayer, err := config.Map("")
```

Comment:
Kept getting errors when I tried to make this an empty dict.Dicter. This is a hack to try to fix that

Reply:
```go
var dict dict.Dict

or 
dict.Dict{}

```